### DISCUSSION TITLE
Be presented with several predefined server destinations at opening

### DISCUSSION THREAD

**Discussion by axlgit** [bauerj/paperless_app#41] [Ideas]
Hi,

would be possible to have the most used servers presented at the beginning? This avoids typing.
Or a list of predefined servers if that makes more sense.
Personally I have two Paperless-ng servers: one for personal use and one for business.

Alex

**Comment by bauerj** [comment]
So are you logging out and then back in every time you switch servers? It works probably be a lot easier to use a "clone app" feature of your OS has that.

**Reply by axlgit** [reply]
Yes, cloning the app id what I would like to avoid. I don't know how much PL-ng app keeps on the phone once disconnected, is it that much? But your suggestion should work too.

**Reply by bauerj** [reply]
Do you mean storage-wise? It should only be a few MB of cached images.

**Reply by axlgit** [reply]
Yes, I just wonder what would be the impact of having two or three servers defined.

**Reply by bauerj** [reply]
Currently, that's not implemented. If you just cloned the app, it would use about twice the storage.
